### ISSUE TITLE
[CoE Starter Kit - BUG] Parse Flow JSON issue with SYNC HELPER - Cloud Flows

### ISSUE THREAD

**Issue by bhavikj65** [microsoft/coe-starter-kit#5996]
### Does this bug already exist in our backlog?

- [X] I have checked and confirm this is a new bug.

### Describe the issue

The Parse Flow JSON action within "SYNC HELPER - Cloud Flows" flow fails for some cloud flows and below is the error message:

![image](https://github.com/microsoft/coe-starter-kit/assets/105588534/8be79214-2848-43ca-a962-d569c79b0173)

InternalServerError. Encountered internal server error. The tracking Id is '02428134-b28a-4e55-86dc-5388f4744cbd'.

I worked with Microsoft Support, and they checked telemetry logs and found below error:

Newtonsoft.Json.JsonReaderException: The reader's MaxDepth of 64 has been exceeded. Path 'properties.definition.actions.Parse_JSON_Tech.inputs.schema.properties.properties.properties.properties.properties.properties.properties.properties.properties.properties.properties.properties.properties.properties.properties.Options.properties.properties.properties.properties.properties.properties.properties.items.properties.properties.properties.properties.properties.properties.properties.properties.properties.properties.properties.properties.properties.properties.properties.Label.properties.properties.properties.properties.properties.properties.properties.properties.properties.properties.properties.properties.properties.properties.properties.LocalizedLabels.properties.type'.

We tested this with a new flow where we got output of Get Flow as Admin and used the same Parse JSON action and found the same error. We then only parsed the actions & triggers as input to the Parse JSON action and that seems to wok fine. 

### Expected Behavior

Parse Flow JSON action within SYNC HELPER - CLoud Flows should work fine. If it fails, I think the error should be handled and maybe Parse JSON action should be optimized by creating multiple actions and passing different properties as inputs for example: 

![image](https://github.com/microsoft/coe-starter-kit/assets/105588534/9e6132ce-8b68-498f-8df7-b34d025993f3)
![image](https://github.com/microsoft/coe-starter-kit/assets/105588534/6b0cc8a4-ca88-46c8-9fae-8b7e1b59b0d0)

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

4.8

### What app or flow are you having the issue with?

SYNC HELPER - Cloud Flows

### What method are you using to get inventory and telemetry?

Cloud flows

### Steps To Reproduce

There is no repro for this as it only seems to fail for cloud flows which returns larger JSON response in Get Flow as Admin action.

**Comment by Jenefer-Monroe** [comment]
We cannot only parse those sections, we need to be able to parse the whole flow, so this is something that we would need to address with the product team. 

Can you confirm:
- You are saying this one repros for a small subset of your flows correct? I have not seen this before so double checking. 
- Are you using some localized labels feature explicitly? Or is that property a mystery to you as well?

**Comment by bhavikj65** [comment]
We are seeing this only for 2 cloud flows so far and I remember the Microsoft Support engineer mentioning about localized labels so I one of this flow has it.  While checking output of the Get Flow as Admin for that affected flow, I could see below as localized labels:

`"Description": {
                                                    "type": "object",
                                                    "properties": {
                                                        "LocalizedLabels": {
                                                            "type": "array",
                                                            "items": {
                                                                "type": "object",
                                                                "properties": {
                                                                    "Label": {
                                                                        "type": "string"
                                                                    },
                                                                    "LanguageCode": {
                                                                        "type": "integer"
                                                                    },
                                                                    "IsManaged": {
                                                                        "type": "boolean"
                                                                    },
                                                                    "MetadataId": {
                                                                        "type": "string"
                                                                    },
                                                                    "HasChanged": {}
                                                                },
                                                                "required": [
                                                                    "Label",
                                                                    "LanguageCode",
                                                                    "IsManaged",
                                                                    "MetadataId",
                                                                    "HasChanged"
                                                                ]
                                                            }`

**Comment by Jenefer-Monroe** [comment]
Chatted with my counterpart and do think we could work around this but it would mean more API calls so want to make sure that woudl be justified here. 
Do you know how the flow got that feature introduced?

**Comment by bhavikj65** [comment]
I am not owner of the cloud flow and hence not sure how that feature got
added there.

If you have specific questions, I could ask the flow owner.

Thank you,
Bhavik

**Comment by Jenefer-Monroe** [comment]
I supposed you can just ask them if they are using some kind of localization feature in the flow.

**Comment by bhavikj65** [comment]
I asked flow owner and he confirmed that he hasn't used any localization features, have asked him to share the flow with me to check.

**Comment by Jenefer-Monroe** [comment]
Placing on backlog for consideration

**Comment by mohamrizwa** [comment]
Hi @bhavikj65, 
Apologies for delayed response.
Please proceed with upgrading to the latest version of the Center of Excellence (CoE) Starter Kit. Before starting, ensure all unmanaged layers are identified and removed to avoid deployment issues. 
For detailed guidance, refer to the below -mentioned 

 official documentation: 
 https://learn.microsoft.com/power-platform/guidance/coe/after-setup 

Feel free to reach out if you need any assistance. We're happy to help!

**Comment by mohamrizwa** [comment]
We haven’t heard back in a while, so we’ll proceed to close this issue for now. If the issue persists or you need further assistance, feel free to a new issue. We’re happy to help anytime. Thanks again for your engagement!

**Comment by CATToolsALMBOT** [comment]
@bhavikj65 This has been fixed in the latest release. Please install [the latest version of the toolkit](https://aka.ms/CoEStarterKitDownload) following the instructions for [installing updates](https://docs.microsoft.com/power-platform/guidance/coe/setup#installing-updates). Note that if you do not remove the unmanaged layers as described there you will not receive updates from us.
